src/libbacktrace/Makefile.am

```diff
@@ -6,12 +6,12 @@
 # met:
 
 #     (1) Redistributions of source code must retain the above copyright
-#     notice, this list of conditions and the following disclaimer. 
```

Comment:
I think that generally, removing it is good, but it can't be fully done with a script, eg, for reasons like both of these.

I agree with @rkruppe about everything in `libbacktrace`. I'll review the rest now.